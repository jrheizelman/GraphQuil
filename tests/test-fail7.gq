/* 
 * Author: Gemma Ragozzine
 * Test formal arguments of main method
 */

void main(int j, int k){
	String a;
}