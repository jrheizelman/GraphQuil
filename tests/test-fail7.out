Fatal error: exception Failure("Main method argument list must be empty")